src/common/gui/SurgeGUIEditor.cpp

```diff
@@ -1268,7 +1268,7 @@ void SurgeGUIEditor::openOrRecreateEditor()
    {
       auto fxpp = currentSkin->getOrCreateControlForConnector("fx.param.panel");
       CRect fxRect = CRect( CPoint( fxpp->x, fxpp->y ), CPoint( 123, 13 ) );
-      for (int i = 0; i < 8; i++)
+      for (int i = 0; i < n_fx_slots; i++)
```

Comment:
This 8 is not n_fx_slots. In fact I'm not entirely sure why it is 8 (other than the only usage of this caps at 3). The size of the array we are writing to is 16 and we write at index + 1 so this should probably be 15.